src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -35,6 +35,12 @@
     private $end;
     private $state;
     private $values;
+    private $enableOverride;
+
+    public function __construct(bool $enableOverride = false)
```

Comment:
Maybe a docblock specifying what the parameter is for.

Reply:
docblock added.